src/ui/atoms/textField/TextField.tsx

```diff
@@ -16,52 +16,28 @@ export type TextFieldProps = InputBaseProps & {
    * Displays a message to the user below the input area.
    */
   readonly helperText?: string
-  /**
-   * If true, the TextField will take 100% of its parent's size.
-   */
-  readonly fullWidth?: boolean
-  /**
-   * If true, the TextField allows multiline text as a text area.
-   */
-  readonly multiline?: boolean
-  /**
-   * If multiline, defines the number of lines of the text area.
-   */
-  readonly numberOfLines?: number
-  /**
-   * If true and multiline, the user can't resize the text area.
-   */
-  readonly disableAreaResize?: boolean
 }
 
 export const TextField: React.FC<TextFieldProps> = ({
   size = 'medium',
   helperText,
-  fullWidth = false,
-  multiline = false,
-  numberOfLines,
-  disableAreaResize = false,
   ...props
 }: TextFieldProps): JSX.Element => {
+  const { multiline, disableAreaResize, fullWidth, hasError }: InputBaseProps = props
```

Comment:
Why not pass these props as arguments?

Reply:
Because we don't want to repeat the props of `InputBase`